cni/network/plugin/main.go

```diff
@@ -224,10 +224,15 @@ func rootExecute() error {
 
 		// Start telemetry process if not already started. This should be done inside lock, otherwise multiple process
 		// end up creating/killing telemetry process results in undesired state.
+		if err = netPlugin.Plugin.LockKeyValueStore(); err != nil {
```

Comment:
lets not do it here..lets move it to a place where we start telemetry service otherwise this lock will be acquired for every cni invocation